### DISCUSSION TITLE
added customization option

### DISCUSSION THREAD

**Discussion by gogo25171** [osmandapp/OsmAnd#13301] [Ideas]
Hello,
I would like you to implement the option to be able to customize position icons.
Also know if you can do a bit like waze, to be able to import files to make any type of voice and logo and then to better personalize the application (but I believe that it is already possible). Because My goal will be to personalize the application by theme as it will have Christmas, and therefore allow me to put as position icons a sleigh with a Santa Claus, and other things.
Thank you in advance, truthfully yours.

**Comment by vshcherb** [comment]
Hello, yes there were some plans to use any possible icon as location icon. We plan to implement it in 2022.
